packages/core/src/components/table/table/test/editing/body-input-wrapper/table.e2e.ts

```diff
@@ -0,0 +1,30 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/table/table/test/editing/body-input-wrapper/index.html';
+
+test.describe.parallel('tds-table-editable-cells', () => {
+  test('renders table with editable cells correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const tableComponent = page.getByRole('table');
+    await expect(tableComponent).toHaveCount(1);
+
+    /* Check diff of screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0.01 });
+  });
+  // test('table has header "Filter"', async ({ page }) => {
```

Comment:
Is this intentionally?

Reply:
no, i copied in to see code in other tab.
Thought i removed that, should not be there

Reply:
no, i copied in to see code in other tab.
Thought i removed that, should not be there

Reply:
no, i copied in to see code in other tab.
Thought i removed that, should not be there